### PR TITLE
Louison branch

### PR DESCRIPTION
**Pull request by LouisonGitzinger**
Ready for the first PR !

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,25 +53,55 @@ To this date there are no shared database. Each contributor has to generate his
...
 
 ## Populating the database
-To this date, we use manually entered information you can find on [this excel file](). To retreive all the data and store it into your db we use the [gspread package](https://github.com/burnash/gspread). This whole part takes plance in the `dbscrap.py` file. 
-For the process to work you will need Google OAuth credentials and download the json file. 
-As the project is still small please [reach out](https://rashadkaranouh.typeform.com/to/yqUf7v) so I can add your credentials to the file before it's made public.
+
+In order to peacefully work locally, you have to import a snapshot of the production database in your local database to populate it.
+Here are the steps to do that : 
+
+Make sure you have have a running local psql database named rapaxademix.
```
Comment (heus92) [inline README.md]: typo on database name it should be *rapacademix* not *rapacademix* 😉

File: README.md [hunk 2]
```diff
@@ -53,25 +53,55 @@ To this date there are no shared database. Each contributor has to generate his
...
+It's time to populate this database with real data.
+Download the snapshot of the production database here : 
+https://drive.google.com/open?id=1byA2yd8C113E9G0hvaeKaCunRZUg3lqW
+
+Then execute the file with psql : 
+```sh
+$ psql rapacademix < path_to_/rapacademix-dbdump.sql
+```
+
+And it done. If everything worked well, you should have a local database ready to use with Django. If something went wrong,
```
Comment (heus92) [inline README.md]: typo *it's done*

File: README.md [hunk 3]
```diff
@@ -53,25 +53,55 @@ To this date there are no shared database. Each contributor has to generate his
...
+Download the snapshot of the production database here : 
+https://drive.google.com/open?id=1byA2yd8C113E9G0hvaeKaCunRZUg3lqW
+
+Then execute the file with psql : 
+```sh
+$ psql rapacademix < path_to_/rapacademix-dbdump.sql
+```
+
+And it done. If everything worked well, you should have a local database ready to use with Django. If something went wrong, 
+don't hesitate to sumbit an issue or reach us.
```
Comment (heus92) [inline README.md]: *reach out tu us*

File: README.md [hunk 4]
```diff
@@ -53,25 +53,55 @@ To this date there are no shared database. Each contributor has to generate his
...
 ## Populating the database
-To this date, we use manually entered information you can find on [this excel file](). To retreive all the data and store it into your db we use the [gspread package](https://github.com/burnash/gspread). This whole part takes plance in the `dbscrap.py` file. 
-For the process to work you will need Google OAuth credentials and download the json file. 
-As the project is still small please [reach out](https://rashadkaranouh.typeform.com/to/yqUf7v) so I can add your credentials to the file before it's made public.
+
+In order to peacefully work locally, you have to import a snapshot of the production database in your local database to populate it.
+Here are the steps to do that : 
+
+Make sure you have have a running local psql database named rapaxademix.
+If not, Launch your `psql` (`$ psql postgres`) command line and enter the following line:
```
Comment (heus92) [inline README.md]: Why aren't the django commands 
```
python manage.py makemigrations
python manage.py migrate
```
not enough ?

File: to_correct [hunk 5]
```diff
@@ -0,0 +1,3 @@
+manage.py in root
```
Comment (heus92) [inline to_correct]: if possible to remove this file from PR could be 👌

File: README.md (hunk 4 above)
Comment (LouisonGitzinger) [reply README.md]: I think you have have to create the DB first by yourself

Comment (heus92) [reply README.md]: not 100% sure of that. I think Django does it 🤔

File: requirements.txt [hunk 6]
```diff
@@ -25,4 +25,4 @@ rsa==3.4.2
 six==1.11.0
 urllib3==1.22
 whitenoise==3.3.1
-django-progressive-web-app==0.1.1
\ No newline at end of file
+django-progressive-web-app==0.1.1
```
Comment (heus92) [inline requirements.txt]: Already exists on row 10 😂

File: searchengine/templates/login.html [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+<h2>Login</h2>
+<form method="post">
+    {% csrf_token %}
+    {{ form.as_p }}
+    <button type="submit">Login</button>
+</form>
+<br>
+<p><strong>-- OR --</strong></p>
+<a href="{% url 'social:begin' 'github' %}">Login with GitHub</a><br>
+{% endblock %}
```
Comment (heus92) [inline searchengine/templates/login.html]: line break missing 😉

File: steps [hunk 8]
```diff
@@ -0,0 +1,12 @@
+1. Create your database in postgres:
```
Comment (heus92) [inline steps]: what's the use of this file ?

File: rapacademix/urls.py [hunk 9]
```diff
@@ -28,6 +28,7 @@
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('searchengine/', include('searchengine.urls')),
+    path('accounts/', include('django.contrib.auth.urls')),
```
Comment (heus92) [inline rapacademix/urls.py]: is it related to accounts / login stuff ?

File: searchengine/templates/home.html [hunk 10]
```diff
@@ -0,0 +1,6 @@
+{% extends 'base.html' %}
```
Comment (heus92) [inline searchengine/templates/home.html]: template related to login ?

File: searchengine/urls.py [hunk 11]
```diff
@@ -1,5 +1,8 @@
-from django.conf.urls import url
+from django.conf.urls import url, include
 from django.urls import path
+from django.contrib import admin
```
Comment (heus92) [inline searchengine/urls.py]: what for ?

File: searchengine/urls.py [hunk 12]
```diff
@@ -1,5 +1,8 @@
...
+from django.contrib.auth import views as auth_views
```
Comment (heus92) [inline searchengine/urls.py]: what for ?

File: searchengine/views.py [hunk 13]
```diff
@@ -3,10 +3,12 @@
 from django.template import RequestContext
 from django.conf import settings
 import random
+from django.contrib.auth.decorators import login_required
```
Comment (heus92) [inline searchengine/views.py]: login stuff ?

File: searchengine/views.py [hunk 14]
```diff
@@ -16,30 +18,37 @@ def index(request):
...
+        context={'num_poa': num_poa,
+                 'num_artists': num_artists,
+                 'num_arts': num_arts,
+                 'num_tags': num_tags,
+                 'algolia_application_id': settings.ALGOLIA['APPLICATION_ID'],
+                 'algolia_search_key': settings.ALGOLIA['SEARCH_KEY'],
+                 'featured_collections': featured_collections}
+    )
+
+@login_required
```
Comment (heus92) [inline searchengine/views.py]: login stuff ?

**Pull request closed without merging**